QUVCView.pro

```diff
@@ -56,7 +56,7 @@ macx {
     ICON = icons/QUVCView.icns
     QMAKE_INFO_PLIST = Info.plist
     CONFIG += app_bundle
-    OPENCV_PATH = /usr/local/Cellar/opencv/4.2.0_1
```

Comment:
this could be automated with `opreadlink -f $(brew --prefix opencv)` or similar to get the local path